pgoapi/pgoapi.py

```diff
@@ -242,11 +259,17 @@ def spin_nearest_fort(self):
         if destinations:
             nearest_fort = destinations[0][0]
             nearest_fort_dis = destinations[0][1]
-            if nearest_fort_dis <= 40.00:
-                self.fort_search_pgoapi(nearest_fort, player_postion=self.get_position(),
-                                        fort_distance=nearest_fort_dis)
-                if 'lure_info' in nearest_fort:
-                    self.disk_encounter_pokemon(nearest_fort['lure_info'])
+            self.log.info('Nearest fort dist %s', nearest_fort_dis)
```

Comment:
This breaks the feature. 
The feature uses Google Maps to calculate the optimal route b/w visible 20 pokestops. google just gives us the route and we have to check every step to see did we encounter a pokestop or not. This will break the google GPS navigation since it will make it walk to a certain fort by force which it could have walked to normally. Sometimes google MAY not  take you within 40 meters of a waypoint But since its experimental, it's supposed to be used in crowded cities and downtown areas where there can be walking directions.